lib/mock-server/pretender-config.js

```diff
@@ -129,7 +131,8 @@ export default class PretenderConfig {
     }
 
     let didOverridePretenderConfig =
-      config.trackRequests !== undefined && this.pretender;
+      config.trackRequests !== undefined &&
+      config.trackRequests !== this.trackRequests;
```

Comment:
So, instead of checking whether there's a `pretender` instance, we look to see whether the `trackRequests` has changed.  This makes sense, although the name of `didOverridePretenderConfig` feels overly broad for what's actually being checked.  But, I think that's fine.

Reply:
Its a local var, I was just changing the minimum